Jenkinsfile

```diff
@@ -57,25 +57,30 @@ node ("ts-engine && heavy && java8") {
         }
     }
 
-    stage('Integration Tests') {
-        try {
-            sh './gradlew --console=plain integrationTest'
-        } finally {
-            junit testResults: '**/build/test-results/integrationTest/*.xml', allowEmptyResults: true
-        }
-    }
-
     stage('Analytics') {
-        sh './gradlew --console=plain check -x test spotbugsmain'
-        recordIssues tool: checkStyle(pattern: '**/build/reports/checkstyle/*.xml')
-        recordIssues tool: spotBugs(pattern: '**/build/reports/spotbugs/main/*.xml', useRankAsPriority: true)
-        recordIssues tool: pmdParser(pattern: '**/build/reports/pmd/*.xml')
-        recordIssues tool: taskScanner(includePattern: '**/*.java,**/*.groovy,**/*.gradle', lowTags: 'WIBNIF', normalTags: 'TODO', highTags: 'ASAP')
+        sh './gradlew --console=plain check -x test'
+        discoverGitReferenceBuild(defaultBranch: 'develop') //TODO: does this also work for PRs with different base branch?
```

Comment:
I think that because we run this under a Jenkins multibranch pipeline, we don't need to specify `defaultBranch` here.

Reply:
Actually, we do! If left empty, it defaults to `master` which is not the right branch to compare against. In particular, our `develop` (and PRs based on that) have diverged more than 100 commits from `master` and the Git Forensics Plugin cannot resolve a common ancestor build.

Reply:
Huh. That's disappointing, as Jenkins multibranch already knows when things are part of a PR, not just some random build happening in a vacuum... oh well. There are very very few PRs based on branches other than develop, so this should be fine.